cli/flox/src/config/features.rs

```diff
@@ -11,6 +11,8 @@ pub struct Features {
     pub search_strategy: SearchStrategy,
     #[serde(default)]
     pub use_catalog: UseCatalog,
+    #[serde(default)]
+    pub sm: bool,
```

Comment:
Suggest not abbreviating to such an extent.  Even knowing what the PR was, I questioned what `sm` was for a moment.  And a new engineer looking at logs of environment settings seeing `_SM` would almost certainly require a search through code or docs. Same for `features.sm` in code.   
`SERVICE_MANAGEMENT` isn't too too long, but, `SERVICES` or `SERVICE_MGMT` would also work and be a bit more concise.